lib/Service/ProjectService.php

```diff
@@ -3613,6 +3616,112 @@ public function cronRepeatBills(?int $billId = null): array {
 		return $result;
 	}
 
+	private function copyBillPaymentMethodOver (string $projectid, array $bill, string $toprojectid): int {
+		$originPayments = $this->getCategoriesOrPaymentModes($projectid, false);
+		$destinationPayments = $this->getCategoriesOrPaymentModes($toprojectid, false);
+
+		if ($bill ['paymentmodeid'] != 0)
+		{
+			$originPayment = array_filter ($originPayments, function ($val) use ($bill) { return $val ['id'] == $bill ['paymentmodeid']; });
```

Comment:
```suggestion
			$originPayment = array_filter($originPayments, static function ($val) use ($bill) {
				return $val ['id'] === $bill ['paymentmodeid'];
			});
```
* no one line
* use `static` for lambda functions
* triple equal

Reply:
👍 Fixed everywhere